src/gui/LocationDialog.cpp

```diff
@@ -291,6 +292,30 @@ void LocationDialog::connectEditSignals()
 	connect(ui->cityNameLineEdit, SIGNAL(textEdited(const QString&)), this, SLOT(reportEdit()));
 }
 
+void LocationDialog::setLocationUIvisible(bool visible)
+{
+	ui->latitudeSpinBox->setEnabled(visible);
```

Comment:
Would be better to just hide the containing widgets like `frame_4`, `frame_2`, and then only the individual controls from `frame_5`. This would simplify the code and avoid the need to revisit it if we add additional elements into these frames.

Reply:
Does the window not collapse if I hide the left-side frame? Then wheeling through the planets would fail.
These changes were now made just to make unfitting controls unavailable without actually changing the layout.

Reply:
I think the other panels should expand into the newly freed space. Needs checking how the wheeling will work though.